patch

```diff
@@ -0,0 +1,2441 @@
+diff --git a/lib/python/rose/apps/ana_builtin/grepper.py b/lib/python/rose/apps/ana_builtin/grepper.py
```

Comment:
A debug file?

Reply:
No, a diff file. I spotted it btween reading Sadie's comments and this... :roll_eyes: